### PR TITLE
Add bazel helper library

### PR DESCRIPTION
**Pull request by achew22**
After the discussion in bazelbuild/rules_webtesting#108 I figured I'd put together a PR to see what people think.

@DrMarcII, what do you think about this? Are you okay with me moving it? Are you okay with me making it //visibility:public?

@jayconrod what do you think about putting it here?

### CODE REVIEW

File: go/tools/bazel/bazel.go [hunk 1]
```diff
@@ -0,0 +1,90 @@
+// Copyright 2017 Google Inc.
```
Comment (jayconrod) [inline go/tools/bazel/bazel.go]: I think this needs to be "The Bazel Authors" instead of Google.

File: go/tools/bazel/bazel_test.go [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+		t.Errorf("Error finding runfiles path: %s", err)
+	}
+
+	if path == "" {
+		t.Errorf("Runfiles path is empty: %s", path)
+	}
+}
+
+func TestNewTmpDir(t *testing.T) {
+	//prefix := "new/temp/dir"
```
Comment (jayconrod) [inline go/tools/bazel/bazel_test.go]: Can this line be removed?

File: go/tools/bazel/bazel_test.go [hunk 3]
```diff
@@ -0,0 +1,75 @@
+// Copyright 2017 Google Inc.
```
Comment (jayconrod) [inline go/tools/bazel/bazel_test.go]: "The Bazel Authors"

**Comment by DrMarcII** [comment]
I would prefer you kept the ability to set the default workspace (maybe return an error from TestWorkspace function if default not set and TEST_WORKSPACE is not defined). When running tests directly from bazel-bin the TEST_WORKSPACE variable doesn't get set.

**Comment by achew22** [comment]
@DrMarcII I have updated the API to support that functionality. PTAL

**Comment by DrMarcII** [comment]
LGTM. Thanks.

**Comment by achew22** [comment]
@DrMarcII any time. Thanks for doing the hard work of actually writing the code :wink: 

@jayconrod, I'm getting CI failures that I don't think are related to this. What would you like to do?

Also, if/when we merge this would you like me to squash it or do you use the GitHub UI to squash?

**Comment by jayconrod** [comment]
Agree the failures aren't due to this change. Travis CI doesn't seem to be running the install script for Bazel on some configurations.

I'll go ahead and submit this now.

**Pull request merged**